--- origsrc/xfe-1.19.1/src/Makefile.am	2008-07-03 06:26:12.000000000 -0500
+++ src/xfe-1.19.1/src/Makefile.am	2008-07-11 19:02:20.671875000 -0500
@@ -30,7 +30,7 @@
 			  TextLabel.cpp \
 			  main.cpp
 
-xfe_LDADD = @LIBINTL@
+xfe_LDADD = $(LTLIBINTL)
 
 xfw_SOURCES = icons.cpp \
               File.cpp \
@@ -51,7 +51,7 @@
 			  FontDialog.cpp \
               XFileWrite.cpp
 			  
-xfw_LDADD = @LIBINTL@
+xfw_LDADD = $(LTLIBINTL)
 
 
 xfp_SOURCES = icons.cpp \
@@ -71,7 +71,7 @@
  			  DirHistBox.cpp \
               XFilePackage.cpp
 
-xfp_LDADD = @LIBINTL@
+xfp_LDADD = $(LTLIBINTL)
 
 xfv_SOURCES = icons.cpp \
               File.cpp \
@@ -92,7 +92,7 @@
 			  FontDialog.cpp \
               XFileView.cpp
 
-xfv_LDADD = @LIBINTL@
+xfv_LDADD = $(LTLIBINTL)
 
 xfi_SOURCES = icons.cpp \
               File.cpp \
@@ -111,7 +111,7 @@
  			  PathLinker.cpp \
               XFileImage.cpp
 				
-xfi_LDADD = @LIBINTL@
+xfi_LDADD = $(LTLIBINTL)
 
 localedir = $(datadir)/locale
 
