--- origsrc/xfe-1.32.1/xvt/ttyinit.c	2009-10-19 04:51:14.000000000 -0500
+++ src/xfe-1.32.1/xvt/ttyinit.c	2009-11-18 22:05:29.262543400 -0600
@@ -155,7 +155,7 @@
 #endif /* HPUX */
 
 /*#ifdef LINUX*/
-#ifdef linux
+#if defined(linux) || defined(__CYGWIN__)
 #include <sys/ioctl.h>
 #include <pty.h>
 #define LIBC_OPENPTY
--- origsrc/xfe-1.32.1/xvt/xvt.h	2009-10-19 04:36:02.000000000 -0500
+++ src/xfe-1.32.1/xvt/xvt.h	2009-11-18 22:06:38.323863800 -0600
@@ -60,7 +60,7 @@ int unlockpt(int);
 char *ptsname(int);
 #else /* !SUNOS5 */
 /*#ifndef LINUX*/
-#ifndef linux
+#if !defined(linux) && !defined(__CYGWIN__)
 int ioctl(int fd, unsigned request, char *arg);
 #endif /* LINUX */
 #endif /* !SUNOS5 */
